lib/sentry.ex

```diff
@@ -338,6 +338,64 @@ defmodule Sentry do
     end
   end
 
+  @doc """
+  Captures a check-in built with the given `options`.
+
+  Check-ins are used to report the status of a monitor to Sentry. This is used
+  to track the health and progress of **cron jobs**. This function is somewhat
+  low level, and mostly useful when you want to report the status of a cron
+  but you are not using any common library to manage your cron jobs.
```

Comment:
So this is basically meant to always be wrapped, right? As 

```
{:ok, check_in_id} = Sentry.capture_check_in(status: :in_progress, monitor_slug: "genserver-job")
```

will break in e.g. testing code if no DSN is set.

Reply:
`capture_check_in/1` will return `:ignored` if DSN is not set, yeah.